src/main/java/lotto/domain/model/Lottos.java

```diff
@@ -0,0 +1,40 @@
+package lotto.domain.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import lotto.domain.Match;
+
+public class Lottos {
+    
+    private final List<Lotto> lottoList;
+    
+    public Lottos(int num) {
+        this(generateLottos(num));
+    }
+    
+    public Lottos(List<Lotto> lottoList) {
+        this.lottoList = lottoList;
+    }
+    
+    public List<Lotto> getLottoList() {
+        return this.lottoList;
+    }
+    
+    private static List<Lotto> generateLottos(int num) {
+        List<Lotto> lottos = new ArrayList<>();
+        for(int i = 0; i < num; i++) {
+            lottos.add(new Lotto());
+        }
+        return lottos;
+    }
+    
+    public WinnerCount findWinners(Lotto winnerLotto) {
+        WinnerCount winnerCount = new WinnerCount();
+        for(Lotto lotto: this.lottoList) {
+            Match match = Match.fromLottoNumber(lotto.findMatchNumbers(winnerLotto.getNumbers()));
```

Comment:
```suggestion
            Match match = lotto.match(winnerLotto);
```
로또의 등수 구하는 로직을 Lotto가 담당하도록 구현하면 어떨까?
객체를 수동적인 존재가 아니라 능동적인 존재로 바라보기 위해 노력해 본다.